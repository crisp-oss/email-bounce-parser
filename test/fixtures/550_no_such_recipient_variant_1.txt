This is the mail system at host mailer.acme.email.

I'm sorry to have to inform you that your message could not
be delivered to one or more recipients. It's attached below.

For further assistance, please send mail to postmaster.

If you do so, please include this problem report. You can
delete your own text from the attached returned message.

                   The mail system

<token@mail.ru>: host mxs.mail.ru[49.101.108.13] said: 550 Message was not
    accepted -- invalid mailbox.  Local mailbox token@mail.ru is
    unavailable: account is disabled (in reply to end of DATA command)
