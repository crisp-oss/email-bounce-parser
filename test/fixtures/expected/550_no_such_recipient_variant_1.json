{
  "bounce": true,
  "email": {
    "body": "This is the mail system at host mailer.acme.email.\n\nI'm sorry to have to inform you that your message could not be delivered to one or more recipients. It's attached below.\n\nFor further assistance, please send mail to postmaster.\n\nIf you do so, please include this problem report. You can delete your own text from the attached returned message.\n\nThe mail system\n\n<token@mail.ru>: host mxs.mail.ru[49.101.108.13] said: 550 Message was not accepted -- invalid mailbox.  Local mailbox token@mail.ru is unavailable: account is disabled (in reply to end of DATA command)",
    "intro": "This is the mail system at host mailer.acme.email.\n\nI'm sorry to have to inform you that your message could not be delivered to one or more recipients. It's attached below.\n\nFor further assistance, please send mail to postmaster.\n\nIf you do so, please include this problem report. You can delete your own text from the attached returned message.\n\nThe mail system",
    "error": "<token@mail.ru>: host mxs.mail.ru[49.101.108.13] said: 550 Message was not accepted -- invalid mailbox.  Local mailbox token@mail.ru is unavailable: account is disabled (in reply to end of DATA command)"
  },
  "data": {
    "error": {
      "code": {
        "basic": "550",
        "enhanced": null
      },
      "label": "Message was not accepted -- invalid mailbox.  Local mailbox token@mail.ru is unavailable: account is disabled",
      "type": "action_not_taken",
      "temporary": false,
      "permanent": true,
      "data": {
        "type": "no_such_recipient",
        "blocked": false,
        "spam": false
      }
    },
    "recipient": "token@mail.ru",
    "server": {
      "hostname": "mxs.mail.ru",
      "ip": "49.101.108.13",
      "port": null
    },
    "command": "DATA"
  }
}